lib/jxl/jxl_test.cc

```diff
@@ -949,6 +949,25 @@ TEST(JxlTest, JXL_SLOW_TEST(RoundtripLosslessNoEncoderFastPathGradient)) {
                                      /*distmap=*/nullptr, &pool));
 }
 
+TEST(JxlTest, JXL_SLOW_TEST(RoundtripLosslessNoEncoderVeryFastPathGradient)) {
+  ThreadPoolInternal pool(8);
+  const PaddedBytes orig =
+      ReadTestData("wesaturate/500px/tmshre_riaphotographs_srgb8.png");
+  CodecInOut io;
+  ASSERT_TRUE(SetFromBytes(Span<const uint8_t>(orig), &io, &pool));
+
+  CompressParams cparams = CParamsForLossless();
+  cparams.speed_tier = SpeedTier::kLightning;
+  cparams.options.skip_encoder_fast_path = true;
```

Comment:
Can we do both?

Reply:
Both with and without skip_encoder_fast_path? Sure.

Reply:
Done.